test/NuGet.Core.FuncTests/NuGet.XPlat.FuncTest/XPlatTestUtils.cs

```diff
@@ -219,14 +219,15 @@ internal static PackageReferenceArgs GetPackageReferenceArgs(string packageId, S
         }
 
         internal static PackageReferenceArgs GetPackageReferenceArgs(string packageId, string packageVersion, SimpleTestProjectContext project,
-            string frameworks = "", string packageDirectory = "", string sources = "", bool noRestore = false, bool noVersion = false, bool prerelease = false)
+            string frameworks = "", string packageDirectory = "", string sources = "", bool noRestore = false, bool noVersion = false, bool prerelease = false, string dgFilePath = "")
```

Comment:
Do you still need this file change?

Reply:
No need, I had some learning curve around Xplat env. I'll do better next time here. Reverted it.